lib/ansible/modules/windows/win_find.ps1

```diff
@@ -250,24 +250,28 @@ Function Get-FileStat($file) {
         try {
             $lnk_source = [Ansible.Command.SymLinkHelper]::GetSymbolicLinkTarget($file)
             $file_stat.lnk_source = $lnk_source
-        } catch {}
+        } catch {
+            #Add-Warning -obj $result -message "symlink source retrieval error: $($_.Exception.Message)"
```

Comment:
Best to keep this the way it was before instead of making a change that would affect the git blame.